src/main/scala/temple/generate/server/go/common/GoCommonGenerator.scala

```diff
@@ -117,26 +117,17 @@ object GoCommonGenerator {
     )
 
   /** Generate function call */
-  private[go] def genFunctionCall(name: String, args: String*): String =
+  private[go] def genFunctionCall(name: String, args: CodeTerm*): String =
     CodeWrap.parens.prefix(name).list(args)
 
   /** Generate function call, but with optional arguments */
   private[go] def genFunctionCall(name: String, firstArg: Option[String], otherArgs: Option[String]*): String =
     CodeWrap.parens.prefix(name).list(firstArg, otherArgs)
 
   /** Generate method call */
-  private[go] def genMethodCall(objectName: String, methodName: String, args: String*): String =
+  private[go] def genMethodCall(objectName: String, methodName: String, args: CodeTerm*): String =
```

Comment:
How come?

Reply:
This was because my last PR #219 had enabled the nesting that was needed here so this could be simplified, and I spotted it when I was checking this directory for renaming